apps/base/workload-identity/workload-identity-ua-identity.yaml

```diff
@@ -6,6 +6,11 @@ metadata:
   annotations:
     serviceoperator.azure.com/reconcile-policy: detach-on-delete
```

Comment:
what if we set this to skip?

Reply:
Hm, potentially.. worth a shot as opposed to tagging all app resources as core i guess - and seems to have same behaviour that deletion in k8s doesn't propagate to Azure, will update